commons/src/main/java/info/metadude/android/eventfahrplan/commons/extensions/StandardExtensions.kt

```diff
@@ -0,0 +1,16 @@
+@file:JvmName("StandardExtensions")
+
+package info.metadude.android.eventfahrplan.commons.extensions
+
+/**
+ * Calls the specified function [block] with `this` value as its argument if `this` is `false`
+ * and returns `this` value.
+ *
+ * Also see [also].
+ */
+inline fun <Boolean> Boolean.onFalse(block: (Boolean) -> Unit): Boolean {
+    if (this == false) {
+        also { block(it) }
```

Comment:
There's no need to use `also` here.

Reply:
I was trying to "extend" on `also` for the special case of `false`.

Reply:
:heavy_check_mark: Removed `also` in a688214fd2582758537df7ef6d9c8c32bbda5e92.